assets/sass/patterns/organisms/listing.scss

```diff
@@ -34,6 +34,21 @@ $totalHighlightsSpacing: 6;
   .content-header--read-more {
     border: none;
   }
+
+  // When wide, hide to avoid duplication: then, the contents is also displayed in the right hand article column
+  .listing-list__item--related:first-child {
```

Comment:
Would that be granular enough? Only one item in the read more list will be a related item, that item is indicated by this `--related` variant.

(It's expected to be the first item., but we could lose the `:first-child` so the related item would still be hidden if it was in a different list position.)